code/game/objects/effects/anomalies/anomalies_transfusion.dm

```diff
@@ -0,0 +1,69 @@
+/obj/effect/anomaly/transfusion
```

Comment:
After looking through this anomaly and its effects I can't say it will be very fun, as blood is pretty expensive as is

Reply:
I'm curious to see how it gets used before calling it. I think there's some potential in the core's detonation and for really bad situations.